com.avaloq.tools.ddk.check.runtime.core/src/com/avaloq/tools/ddk/check/runtime/issue/DefaultCheckImpl.java

```diff
@@ -112,10 +112,24 @@ private void initCheckMethodCache() {
    *          the context
    * @return true, if successful
    */
-  protected final boolean internalValidate(final EClass class1, final EObject object, final DiagnosticChain diagnostics, final Map<Object, Object> context) {
+  protected final boolean internalValidate(final EClass class1, final EObject object, final DiagnosticChain diagnostics, final Map<Object, Object> context) { // NOPMD

```

Comment:
what is the warning we are suppressing with NOPMD?

Reply:
NPathComplexity